sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -235,6 +223,31 @@ private Object[][] gatewayAndDirect() {
         };
     }
 
+    @Test(groups = {"fast"}, timeOut = TIMEOUT)
+    public void queryChangeFeed() throws Exception {
```

Comment:
This should be targeted to the `emulator` group - the accounts created in the live test pipeline do not have support for all versions and deletes. Also, this test can run for incremental change feed pull model as well.

Reply:
fixed